### PR TITLE
[WEBDEV-40] Defining new buttons

### PR DESCRIPTION
**Pull request by UsmanAfzal**

### CODE REVIEW

File: src/stylesheets/components/_buttons.scss [hunk 1]
```diff
@@ -1,103 +1,67 @@
...
+	@include button($grey-3, $grey-4, $grey-4);
 }
 
-// Button with animated loading icon and text
+/*--- Button Loader ---*/
+
 .btn--loading {
-  cursor: not-allowed;
 	height: auto;
+	cursor: not-allowed;
```
Comment (jakemulley) [inline src/stylesheets/components/_buttons.scss]: Not-allowed not supported in IE - just a heads up!

File: src/stylesheets/components/_buttons.scss [hunk 2]
```diff
@@ -1,103 +1,67 @@
-// Button base rules
+/*--- Button Base ---*/
+
 .btn, a.btn {
```
Comment (naseberry) [inline src/stylesheets/components/_buttons.scss]: is 'a.btn' required?

File: src/stylesheets/components/_buttons.scss [hunk 3]
```diff
@@ -1,103 +1,67 @@
...
+	}
 	@include media($desktop) {
 		.form-group & {
 			padding: $base-spacing-unit-tiny $base-spacing-unit-extra-large;
 		}
 	}
 	&:focus, &:active {
-		opacity: 0.6;
 		outline: none;
 		border-radius: 4px;
```
Comment (naseberry) [inline src/stylesheets/components/_buttons.scss]: same as the default state, is this required?

File: src/stylesheets/components/_buttons.scss [hunk 4]
```diff
@@ -1,103 +1,67 @@
...
-		$white 			// a:hover color
-	);
-  color: $white;
-  background-color: $blue;
+/*--- Button Options [colour, background, background-hover] ---*/
 
-  &:hover {
-    background-color: (darken($blue, 10%));
-  }
+.btn--primary, a.btn--primary {
```
Comment (naseberry) [inline src/stylesheets/components/_buttons.scss]: is 'a.btn--primary' required?

File: src/stylesheets/components/_buttons.scss [hunk 5]
```diff
@@ -1,103 +1,67 @@
...
+	@include button($white, $button--primary, $button--primary__hover);
 }
-
 .btn--secondary, a.btn--secondary {
```
Comment (naseberry) [inline src/stylesheets/components/_buttons.scss]: same as above

File: src/stylesheets/components/_buttons.scss [hunk 6]
```diff
@@ -1,103 +1,67 @@
...
-		$white, 		// a:visited color
-		$navy 			// a:hover color
-	);
-	color: $navy;
-	background-color: $secondary;
-
-	&:hover {
-		background-color: (darken($secondary, 10%));
-	}
+.btn--deep-purple, a.btn--deep-purple {
```
Comment (naseberry) [inline src/stylesheets/components/_buttons.scss]: same as above

File: src/stylesheets/components/_buttons.scss [hunk 7]
```diff
@@ -1,103 +1,67 @@
...
+	@include button($white, $button--deep-purple, $button--deep-purple__hover);
+}
+.btn--white, a.btn--white {
```
Comment (naseberry) [inline src/stylesheets/components/_buttons.scss]: as above

File: src/stylesheets/components/_buttons.scss [hunk 8]
```diff
@@ -1,103 +1,67 @@
...
+	@include button($deep-purple, $button--white, $button--white__hover);
+}
+.btn--disabled, a.btn--disabled {
```
Comment (naseberry) [inline src/stylesheets/components/_buttons.scss]: as above

**Pull request merged**